config/jenkins_weekly.ci.jenkins.io.yaml

```
@@ -121,6 +121,9 @@ controller:
           defaultFolderConfiguration:
             # Keep healthMetrics an empty list to ensure weather is disabled
             healthMetrics: []
+          theme-manager:
```

https://github.com/jenkinsci/theme-manager-plugin/blob/280bd2f83dc695f11c4b754f2dd9888cfdb9dacc/src/main/java/io/jenkins/plugins/thememanager/ThemeManagerPageDecorator.java#L21
```suggestion
          themeManager:
```

---

Oh I could swear I copied that